test/bin/not_passed.js

```diff
@@ -8,9 +8,12 @@ plzPort().then((port) => {
   const proc = cp.exec(`node ${process.cwd()}/bin/agreed-server.js --port ${port} --path ${not_pass}`);
   setTimeout(() => {
     const result = cp.execSync(`node ${process.cwd()}/bin/agreed-client.js --port ${port} --path ${pass}`).toString();
+
     console.log(result);
     assert(result.indexOf('✗ fail') >= 0);
     proc.kill();
-  }, 100);
+
+    process.exit(0);
```

Comment:
you would be better to kill process more softly.

according to this PR, we should follow this.

https://github.com/angularity/node-angularity/commit/5bdcedf43dd4bc70ae0a7fb05dad4610f3394b5c#diff-51bec0f0addeb2b05eace161e5e29704R390